Sleep_stage_classifier/shhs/polysomnography/polysomnography_reader.py

```diff
@@ -0,0 +1,107 @@
+import mne
+from shhs.parser import xml_nsrr as xn

```

Comment:
you should be able to do this without copying the same files into your own submodule, referencing the original files instead. We def don't want duplicate files across this